.cicd/build-scripts.yml

```diff
@@ -189,4 +202,20 @@ steps:
     command:
       - "apt update && apt upgrade -y && apt install -y git"
       - "./scripts/eosio_build.sh -y"
-    timeout: 180
\ No newline at end of file
+    timeout: 180
+
+  - label: ":ubuntu: Ubuntu 20.04 - Build UnPinned"
+    env:
+      DEBIAN_FRONTEND: "noninteractive" 
+      TZ: "America/New York"
```

Comment:
This `TZ` variable is missing from the pinned build, but the symbolic link is created in both. Does this variable need to be added there, too? Does the symbolic link make it unnecessary, and can it be removed here?